hnn_core/cell.py

```diff
@@ -153,6 +153,70 @@ def gid(self, gid):
             raise RuntimeError('Global ID for this cell already assigned!')
 
 
+class Section:
+    """Section class.
+
+    Parameters
+    ----------
+    L : float
+        length of a section in microns
+    diam : float
+        diameter of a section in microns
+    cm : float
+        membrane capacitance in micro-Farads
+    Ra : float
+        axial resistivity in ohm-cm
+    end_pts : list of [x, y, z]
+        The start and stop points of the section.
+
+    Attributes
+    ----------
+    mechs : dict
+        Mechanisms to insert in this section. The keys
+        are the names of the mechanisms and values
+        are the properties. For e.g., {'ca': {'gbar_ca': 60}}
+    syns : list of str
+        The synaptic mechanisms to add in this section
+    end_pts : list of [x, y, z]
+        The start and stop points of the section. Cannot be changed.
+    L : float
+        length of a section in microns. Cannot be changed.
+    diam : float
+        diameter of a section in microns. Cannot be changed.
```

Comment:
Is the only reason for making these private to prevent users from trying to plot them given the issue raised by @ntolley? If so, it just occurred to me that maybe we should instead just prevent (or warn) users from running `plot_cell_morphology()` on the template cells prior to running the simulation because the length and radius haven't been used to update the end points for any of the cells yet. Upon running the simulation, we could then just copy over the properties from one of the network cells to the template cell for each type.